yaml-tests/src/test/resources/groupby-tests.yamsql

```diff
@@ -154,127 +118,34 @@ test_block:
-      # Plans did not serialize before 4.1.9.0 due to: https://github.com/FoundationDB/fdb-record-layer/issues/3214
-      - supported_version: 4.1.9.0
       - result: [{!l 5}, {!l 13}]
-    -
-      # Same as above but simulates force_continuations mode. Can be removed after we no longer care about compatibility with versions before 4.1.9.0
-      - query: select MAX(x.col2) from (select col1,col2 from t1) as x group by x.col1;
-      - maxRows: 1
-      - initialVersionLessThan: 4.1.9.0
-      - result: [{!l 5}]
-      - result: [{!l 13}]
-      - error: 'XX000'
-      - initialVersionAtLeast: 4.1.9.0
     -
       - query: select MIN(x.col2) from (select col1,col2 from t1) as x group by x.col1;
-      # Plans did not serialize before 4.1.9.0 due to: https://github.com/FoundationDB/fdb-record-layer/issues/3214
-      - supported_version: 4.1.9.0
       - result: [{!l 1}, {!l 6}]
-    -
-      # Same as above but simulates force_continuations mode. Can be removed after we no longer care about compatibility with versions before 4.1.9.0
-      - query: select MIN(x.col2) from (select col1,col2 from t1) as x group by x.col1;
-      - maxRows: 1
-      - initialVersionLessThan: 4.1.9.0
-      - result: [{!l 1}]
-      - result: [{!l 7}] # Off by one from https://github.com/FoundationDB/fdb-record-layer/issues/3097 also fixed in 4.1.9.0
-      - error: 'XX000'
-      - initialVersionAtLeast: 4.1.9.0
     -
       - query: select COUNT(x.col2) from (select col1,col2 from t1) as x group by x.col1;
-      # Plans did not serialize before 4.1.9.0 due to: https://github.com/FoundationDB/fdb-record-layer/issues/3214
-      - supported_version: 4.1.9.0
       - result: [{!l 5}, {!l 8}]
-    -
-      # Same as above but simulates force_continuations mode. Can be removed after we no longer care about compatibility with versions before 4.1.9.0
-      - query: select COUNT(x.col2) from (select col1,col2 from t1) as x group by x.col1;
-      - maxRows: 1
-      - initialVersionLessThan: 4.1.9.0
-      - result: [{!l 5}]
-      - result: [{!l 7}] # Off by one from https://github.com/FoundationDB/fdb-record-layer/issues/3097 also fixed in 4.1.9.0
-      - error: 'XX000'
-      - initialVersionAtLeast: 4.1.9.0
     -
       - query: select AVG(x.col2) from (select col1,col2 from t1) as x group by x.col1;
       - explain: "ISCAN(I1 <,>) | MAP (_.COL1 AS COL1, _.COL2 AS COL2) | MAP (_ AS _0) | AGG (avg_l(_._0.COL2) AS _0) GROUP BY (_._0.COL1 AS _0) | MAP (_._1._0 AS _0)"
-      - maxRows: 0 # Disable force_continuations. Can be removed when we longer care about testing with versions older than 4.1.9.0
-      - result: [{3.0}, {9.5}]
-    -
-      # Same as above but simulates force_continuations mode. Can be removed after we no longer care about compatibility with versions before 4.1.9.0
-      - query: select AVG(x.col2) from (select col1,col2 from t1) as x group by x.col1;
       - maxRows: 1
```

Comment:
I think we want to take out the `maxRows: 1` case here in favor of just leaving in the full result and relying on `FORCE_CONTINUATIONS` mode for the max-rows-1 case

Reply:
good catch! removed.